jps-plugin/src/com/goide/jps/builder/GoBuilder.java

```diff
@@ -63,7 +66,14 @@ public void build(@NotNull GoTarget target,
     JpsTypedModule<JpsSimpleElement<JpsGoModuleProperties>> module = jpsModule.asTyped(JpsGoModuleType.INSTANCE);
     assert module != null;
     JpsSdk<JpsDummyElement> sdk = getSdk(context, module);
-    File executable = GoEnvironmentUtil.getExecutableForSdk(sdk.getHomePath());
+    File exec = GoEnvironmentUtil.getExecutableForSdk(sdk.getHomePath());
+    if (exec == null) {
+      Notifications.Bus.notify(
```

Comment:
GoEnvironmentUtil is ok for that. 
Also, as I see all clients of getExecutablePath method show these notifications, so you can add parameter to getExecutablePath like (showNotificationOnFail) and move showing notifications there.